app/assets/stylesheets/layout_components/card.scss

```diff
@@ -0,0 +1,41 @@
+.card {
+  background: $white;
+  padding: $base-padding 0;
```

Comment:
do we need to zero out the left and right padding?

Reply:
actually, can this just be padding-top? the child elements all have bottom padding or margin on them, so this way we wont have to add mb0 to the last child

Reply:
They are set to 0 because of how I was formatting it. I could do padding-top: $base-padding and padding-bottom: $base-padding, would you prefer that?

Reply:
yes, otherwise if we're manually setting/resetting padding and margin it messes with the rows and columns